package.json

```diff
@@ -36,26 +36,25 @@
     "version": "npm run build && git add -A dist"
   },
   "dependencies": {
-    "es6-error": "^3.2.0",
-    "es6-promise": "^4.0.4",
-    "kinvey-node-sdk": "3.3.5",
-    "lodash": "^4.16.2",
-    "parse-headers": "^2.0.1",
-    "rxjs": "5.0.0-beta.12",
```

Comment:
RxJS is a dependency of the core and not Titanium SDK. The reference you see in webpack.config.js is a local file to fix the issue with getting the root. See https://github.com/Kinvey/titanium-sdk/blob/ef8f07d77bc212d8da7d6b277caf887a96189886/src/rxjs.js.